server/testdriver/core/src/main/java/org/infinispan/server/test/core/AbstractInfinispanServerDriver.java

```diff
@@ -84,22 +91,32 @@ public void prepare(String name) {
          throw new RuntimeException("Failed to create server configuration directory " + confDir);
       }
       URL configurationFileURL;
-      if (new File(configuration.configurationFile()).isAbsolute()) {
+      if (DEFAULT_INFINISPAN_CONFIG_FILES.contains(configuration.configurationFile())) {
```

Comment:
Is this change really necessary?

"infinispan.xml" is not an absolute path, so it would use the `else` branch, which looks exactly the same.

Reply:
it raises an exception

Reply:
Are you sure? The code looks exactly the same to me.

Reply:
new File(configuration.configurationFile()) -> raises exception for the file when it is in the jar

Reply:
but the code can be refactored for sure to avoid duplication

Reply:
It shouldn't get to `new File(configuration.configurationFile())`, because `new File(configuration.configurationFile()).isAbsolute()` should be false.

BTW how are you testing this?

Reply:
in the playground project for now of tristan